tinytroupe/extraction/results_extractor.py

```
@@ -0,0 +1,268 @@
+import os
+import json
+import chevron
+import pandas as pd
```

Ruff (0.8.2)

`pandas` imported but unused

Remove unused import: `pandas`

(F401)